src/api/app/controllers/concerns/webui/ajax_datatables_error_handling.rb

```diff
@@ -0,0 +1,14 @@
+# This module is responsible of the error handling of the AjaxDatatables we use in:
+# - Projects subprojects page
+# - Project's package list
+# - Manage users in Configuration page
+# - Filelist of the details page of a binary
+module Webui::AjaxDatatablesErrorHandling
```

Comment:
If you add this to `Webui::RescueHandler` you don't have to include it into individual controllers....

Reply:
I will address this

Reply:
done